.gitignore

```diff
@@ -8,3 +8,6 @@ npm-debug.log
 .jshintrc
 .vs/
 test/unit/three.*.unit.js
+build/three.module.js
```

Comment:
Why do you modify `.gitignore`?

Reply:
I have dicarded changes in .gitignore

Reply:
Um, it seems the file is still modified...